### PR TITLE
#430: Update dependencies

### LINKED ISSUE #430: Update dependencies
**Tailor Version:** v0.10.0

**Swift Version:** 2.2

### PR DESCRIPTION
**Pull request by adityatrivedi**
Resolves #430.

### CODE REVIEW

**Comment by nehasingh2293** [comment]
👍

File: build.gradle [hunk 1]
```diff
@@ -175,7 +175,7 @@ task gems(type: Exec) {
     workingDir 'src/dist/gems'
     executable 'sh'
     // HACK: Workaround for bundler/bundler#4494
```
Comment (alykhank) [inline build.gradle]: If this version fixes the Bundler issue you can remove all instances where the workaround was used along with the comments. I believe previously we did not specify the version explicitly, opting to just use the latest in all cases.

File: build.gradle [hunk 2]
```diff
@@ -174,8 +174,6 @@ task gems(type: Exec) {
     outputs.dir file('src/dist/gems/vendor/')
     workingDir 'src/dist/gems'
     executable 'sh'
-    // HACK: Workaround for bundler/bundler#4494
-    args '-c', 'bundle _1.11.2_ package --no-install'
```
Comment (alykhank) [inline build.gradle]: This can't be removed entirely, it has to be reverted to the version prior to 4c43460d8e45e3b263b9bdf8148ce12c4b215027.

**Comment by alykhank** [comment]
:+1:

**Pull request merged**